packages/eslint-plugin/src/rules/naming-convention-utils/schema.ts

```diff
@@ -290,6 +290,24 @@ const SCHEMA: JSONSchema.JSONSchema4 = {
         'override',
         'async',
       ]),
+      ...selectorSchema('classicAccessor', true, [
+        'abstract',
+        'private',
+        'protected',
+        'public',
+        'requiresQuotes',
+        'static',
+        'override',
+      ]),
+      ...selectorSchema('autoAccessor', true, [
+        'abstract',
+        'private',
+        'protected',
+        'public',
+        'requiresQuotes',
+        'static',
+        'override',
+      ]),
```

Comment:
From https://github.com/typescript-eslint/typescript-eslint/pull/8084#discussion_r1451125434:

> EG like:
> classicAccessor = just the top-level, non-keyworded accessor syntax
> autoAccessor = the new keyworded syntax
> accessor = group of classicAccessor | autoAccessor

So I'd think `autoAccessor` would have just the new keyworded syntax, _not_ the old keywords, no?

Reply:
`abstract`, `private`, `static` -- they all are 'modifiers'.

A 'selector' has multiple modifiers under it.

`autoAccessor` is an 'individual selector'(same as `classicAccessor`)

`accessor` is a 'group selector'. 

& I did that because ... say the modifiers which are applicable to `get` syntax, shouldn't they be also applicable to the `accessor` syntax ? 

`static accessor` is a thing ig ? https://github.com/tc39/proposal-decorators/blob/master/README.md#class-auto-accessors , see 3rd code snippet.